.gitignore

```diff
@@ -1,5 +1,42 @@
-*.swp
-.DS_Store
+# Exercism project
 bin/configlet
 bin/configlet.exe
-*.dproj
```

Comment:
@kotp I think *.dproj should be retained in the .gitignore.  Delphi will generate a new one when the .DPR file is opened by Delphi.